roles/designer.md

```diff
@@ -31,27 +24,12 @@ These are the stages that the designers follow to create a new application or we
 
 ### Wireframe
 
-Product Owner and Marketing talk about the goals of the application or website. They will outline the first content and explain the requirements to the designers and the creation of a wireframe model will be the result.
-
-### Mockup
-
-A mood board is created to enable collaborating on the style of the product. Once a consensus is reached, the details of the wireframe and mood board and captured in a graphical design mockup.
-
-The final version will be approved by the Product Owner and the process moves onto implementing the design.
-
-### Implement
+Product Owner and Marketing talk about the goals of the application or website. They will outline the first content and explain the requirements to the designers and the creation of a wireframe model will be the result (a monochrome sketch). A Mockup is a wireframe with more detail and is only done when such detail is required (usually for sales pages).
 
-The implementation stage is where the actual website or application front-end is created in HTML/CSS/JS using [Boostrap framework](https://getbootstrap.com/).
+### Design Mockup
 
-The final version will be approved by the assigned developer.
+A mood board (a collage consisting of images and text) is created to enable collaborating on the style of the product. Once a consensus is reached, the details of the wireframe and mood board and captured in a graphical design mockup. The final version will be approved by the Product Owner and the process moves onto implementing the design.
```

Comment:
1. `collaborating` -> `collaboration`

2. >Once a consensus is reached, the details of the wireframe and mood board and captured in a graphical design mockup. 

`...are captured in a`

3. >The final version will be approved by the Product Owner and the process moves onto implementing the design.

Wrong use of onto. Use the separate `on to`.